gap/small.gd

```diff
@@ -72,14 +72,48 @@ UnbindGlobal( "SMALL_AVAILABLE" );
 ##  <Func Name="SMALL_AVAILABLE" Arg='order'/>
 ##
 ##  <Description>
-##  returns fail if the library of groups of <A>order</A> is not installed. 
-##  Otherwise a record with some information about the construction of the 
-##  groups of <A>order</A> is returned.
+##  returns fail if the library of groups of order <A>order</A> is not installed.
+##  Otherwise a record with some information about the construction of the
+##  groups of order <A>order</A> is returned.
 ##  </Description>
 ##  </ManSection>
 ##
 DeclareGlobalFunction( "SMALL_AVAILABLE" );
 
+#############################################################################
+##
+#F  SmallGroupsAvailable( <order> )
+##
+##  <#GAPDoc Label="SmallGroupsAvailable">
+##  <ManSection>
+##  <Func Name="SmallGroupsAvailable" Arg='order'/>
+##
+##  <Description>
+##  returns <C>true</C> if the library of groups of order <A>order</A> is
+##  installed, and <C>false</C> otherwise.
+##  </Description>
+##  </ManSection>
+##  <#/GAPDoc>
+##
+DeclareGlobalFunction( "SmallGroupsAvailable" );
+
+#############################################################################
+##
+#F  NumberSmallGroupsAvailable( <order> )
```

Comment:
I wonder if this (and the other ones) shouldn't perhaps instead be called `HasNumberSmallGroups`?